packages/terra-date-input/src/DateInputField.module.scss

```
@@ -1,3 +1,7 @@
+// Themes
+@import './orion-fusion-theme/DateInput.module';
+@import './clinical-lowlight-theme/DateInput.module';
```

```suggestion
@import './orion-fusion-theme/DateInputField.module';
@import './clinical-lowlight-theme/DateInputField.module';
```

These theme vars should be in their own file

---

Done acdc03d